test/src/executioners/SteadyWithNull.C

```diff
@@ -0,0 +1,39 @@
+/****************************************************************/
+/*               DO NOT MODIFY THIS HEADER                      */
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*           (c) 2010 Battelle Energy Alliance, LLC             */
+/*                   ALL RIGHTS RESERVED                        */
+/*                                                              */
+/*          Prepared by Battelle Energy Alliance, LLC           */
+/*            Under Contract No. DE-AC07-05ID14517              */
+/*            With the U. S. Department of Energy               */
+/*                                                              */
+/*            See COPYRIGHT for full restrictions               */
+/****************************************************************/
+#include "SteadyWithNull.h"
+#include "NonlinearSystem.h"
+#include "AuxiliarySystem.h"
+
+template<>
+InputParameters validParams<SteadyWithNull>()
+{
+  InputParameters params = validParams<Steady>();
+  return params;
+}
+
+SteadyWithNull::SteadyWithNull(const InputParameters & parameters) :
+    Steady(parameters)
+{
+}
+
+void
+SteadyWithNull::init()
+{
+  Steady::init();
+  NumericVector<Number> * to_vector = &_problem.getNonlinearSystem().getVector("NullSpace_0");
+  const NumericVector<Number> * from_vector = &_problem.getAuxiliarySystem().solution();
+  *to_vector = *from_vector;
```

Comment:
Do you mean value by value? Isn't this just taking the content of one vector and assigns it to the other?